index.html

```diff
@@ -11,7 +11,30 @@
 </head>
 
 <body>
-  <header class="page-header"></header>
+  <header class="page-header">
+    <div class="header-logo">
+      <a href="#"><img alt="Callium" src="img/Subtract.png"></a>
+    </div>
+    <ul class="header-nav">
+      <li>
+        <a href="#">мебель</a>
+      </li>
+      <li>
+        <a href="#">освещение</a>
+      </li>
+      <li>
+        <a href="#">текстиль</a>
+      </li>
+    </ul>
+    <ul class="user-widgets">
+      <li>
+        <a href="#">Поиск</a>
+      </li>
+      <li>
+        <a href="#">Корзина</a>
+      </li>
+    </ul>
```

Comment:
Навигация по секция сайта должна быть обёрнута в тег [`nav`](https://developer.mozilla.org/ru/docs/Web/HTML/Element/nav)
```suggestion
   <nav>
    <ul class="header-nav">
      <li>
        <a href="#">мебель</a>
      </li>
      <li>
        <a href="#">освещение</a>
      </li>
      <li>
        <a href="#">текстиль</a>
      </li>
    </ul>
   </nav>
```